pyqtgraph/graphicsItems/PlotDataItem.py

```diff
@@ -62,7 +62,7 @@ class PlotDataset:
     
     After a search has been performed, typically during a call to
     :func:`dataRect() <pyqtgraph.PlotDataset.dataRect>`, ``dataset.containsNonfinite``
-    is ``True`` if any coordinate values are non-finite (e.g. NaN or inf) or ``False``
+    is ``True`` if any coordinate values are non-finite (e.g. ``NaN`` or ``Inf``) or ``False``
```

Comment:
I keep going back and forth on this, but should we replace usages of `NaN` with `np.nan` and `np.inf`  for `Inf`?

Reply:
The argument for ``NaN`` is that
- it is more generic
- data input is not necessarily in numpy format.

When *we* generate a NaN, it should probably be `np.nan`, but I think that does not occur here as far as the documentaiton is concerned.